packages/components/src/Popover/Popover.module.css

```diff
@@ -35,47 +35,28 @@
 }
 
 .arrow {
-  visibility: hidden;
-}
-
-.arrow,
-.arrow::before {
```

Comment:
testing it out the arrow is still doing what it is supposed to do and looks good.

what changed though? do we not need to build our our `::before` styles? or were they never necessary?